### ISSUE TITLE
Is it worthwhile to include the Bellcore and ad-hoc licenses

### ISSUE THREAD

**Issue by jeremiah** [mhucka/SoManyLicenses#4]
These licenses are pretty obscure. So far, I've only seen them applied to one project called "mime-support"; 
https://tracker.debian.org/media/packages/m/mime-support/copyright-3.62

Debian's popularity contest says it is pretty widely used: https://qa.debian.org/popcon.php?package=mime-support 
It is part of the perl-base package which is in most Debian and Ubuntu systems.

I cannot find either licenses in the SPDX list "d0f6824" which I believe is the latest version.

**Comment by mhucka** [comment]
Yeah, I agree the Bellcore & other license are not worth it. Thank you for filing this issue.

Originally the goal was to be very complete, but the project that motivated this repo has ended and to be honest I don't have time to pursue more. So, unless you have a strong need for it, I'm going to archive this repo on Dec. 31.
